src/api/public/apidocs-new/components/parameters/user_login.yaml

```diff
@@ -0,0 +1,7 @@
+in: path
+name: user_login
+schema:
+  type: string
+required: true
+description: User login.
```

Comment:
User or person? We should make up our mind.

Reply:
As commented in [PR 10661](https://github.com/openSUSE/open-build-service/pull/10661#discussion_r561121929), I'm going to use "person" and "people" and to rename the parameter and the file to `login`.

Reply:
Done, @hennevogel